starcounter-include.html

```diff
@@ -210,35 +208,67 @@
                 this.template.model = this.partial
             }
         };
+
+        /**
+         * Stamps composition if needed
+         * @param  {String|DocumentFragment} compositionRef reference to the evaluated composition
+         * @param  {Function} callb Function that returns a composition to render
+         */
+        StarcounterIncludePrototype._renderCompositionChange = function(compositionRef, callb) {
+            if (!this._forceLayoutChange && this._lastCompositionRef == compositionRef) {
+                return;
+            }
```

Comment:
Wouldn't it be cleaner to use the desired condition rather than the opposite with a return? E.g:
```js
 if (this._forceLayoutChange || this._lastCompositionRef !== compositionRef) {
    this.storedLayout = null;
    let composition = callb();
    this.stampComposition(composition);
    this._forceLayoutChange = false;
    this._lastCompositionRef = compositionRef;
}
```